ROADMAP.md

```diff
@@ -0,0 +1,76 @@
+# KitOps Roadmap
+
+As a public community project the KitOps roadmap is always open to feedback. If you think a feature lower in the list is more important, or feel that we've missed an important feature please:
+
+* [Open an issue](https://github.com/jozu-ai/kitops/issues) in the repo
+* Tell us in the [KitOps Discord](https://discord.gg/Tapeh8agYy) channel
```

Comment:
Not saying this is going to happen again but we should consider making a note of everywhere we surface the discord invite link in case it expires again and needs to be updated.

Reply:
Good idea, I'll create another pull request to add a discord tracking MD into the repo.